addons/sale_pdf_quote_builder/static/src/binary_sale_pdf_quote/binary_field.js

```diff
@@ -0,0 +1,15 @@
+/** @odoo-module **/
+
+import { registry } from "@web/core/registry";
+import { BinaryField, binaryField } from "@web/views/fields/binary/binary_field";
+
+export class BinarySalePDFQuote extends BinaryField {}
```

Comment:
Waiting for https://github.com/odoo/odoo/pull/160070
As I don't think adding specific code to be able to download pdf file because we might need binary field download option in settings in other place.
Also it is now working because `res.config.settings` does not contains resID that's why it is not showing and not working properly

So waiting for Framework team!